ui/proposal_details_vote_modal.go

```diff
@@ -0,0 +1,260 @@
+	wrap := cm.theme.Card()
+	wrap.Color = wdg.background
+	return wrap.Layout(gtx, func(gtx C) D {
+		inset := layout.Inset{
+			Top:    values.MarginPadding8,
+			Bottom: values.MarginPadding8,
+			Left:   values.MarginPadding16,
+			Right:  values.MarginPadding8,
+		}
+		return inset.Layout(gtx, func(gtx C) D {
+			return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+				layout.Flexed(.4, func(gtx C) D {
+					return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+						layout.Rigid(func(gtx C) D {
+							icon := cm.icons.imageBrightness1
+							icon.Color = cm.theme.Color.Success
+							return icon.Layout(gtx, values.MarginPadding8)
+						}),
+						layout.Rigid(func(gtx C) D {
+							return layout.Inset{Left: values.MarginPadding5}.Layout(gtx, func(gtx C) D {
+								label := cm.theme.Body2(wdg.label)
+								return label.Layout(gtx)
+							})
+						}),
+					)
+				}),
+				layout.Flexed(.6, func(gtx C) D {
+					border := widget.Border{
+						Color:        cm.theme.Color.Gray1,
+						CornerRadius: values.MarginPadding8,
+						Width:        values.MarginPadding2,
+					}
+
+					return border.Layout(gtx, func(gtx C) D {
+						card := cm.theme.Card()
+						card.Color = cm.theme.Color.Surface
+						return card.Layout(gtx, func(gtx C) D {
+							var height int
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return layout.Flex{Alignment: layout.Middle}.Layout(gtx,
+								layout.Flexed(1, func(gtx C) D {
+									dims := layout.Flex{Alignment: layout.Middle, Spacing: layout.SpaceBetween}.Layout(gtx,
+										layout.Rigid(func(gtx C) D {
+											return wdg.dec.Layout(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											gtx.Constraints.Min.X, gtx.Constraints.Max.X = 100, 100
+											return wdg.input.Layout(gtx)
+										}),
+										layout.Rigid(func(gtx C) D {
+											return wdg.inc.Layout(gtx)
+										}),
+									)
+									height = dims.Size.Y
+									return dims
+								}),
+								layout.Flexed(0.02, func(gtx C) D {
+									line := cm.theme.Line(height, gtx.Px(values.MarginPadding2))
+									line.Color = cm.theme.Color.Gray1
+									return line.Layout(gtx)
+								}),
+								layout.Rigid(func(gtx C) D {
+									return wdg.max.Layout(gtx)
+								}),
+							)
+						})
+					})
+				}),
+			)
+		})
+	})
+}
+
+func (i *inputVoteOptionsWidgets) handler() {
+	if i.inc.Button.Clicked() {
+		value, err := strconv.Atoi(i.input.Editor.Text())
+		if err != nil {
+			log.Error(err)
+			return
+		}
+		value++
+		i.input.Editor.SetText(fmt.Sprintf("%d", value))
+	}
+
+	if i.dec.Button.Clicked() {
+		value, err := strconv.Atoi(i.input.Editor.Text())
+		if err != nil {
+			log.Error(err)
+			return
+		}
+		value--
+		if value < 0 {
+			return
+		}
+		i.input.Editor.SetText(fmt.Sprintf("%d", value))
+	}
+
+	if i.max.Button.Clicked() {
+		i.input.Editor.SetText("5")
+	}
+}
```

Comment:
This method can be above the `handle` method and renamed to `handleVoteCountButtons`